### PR TITLE
Java cc12

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/FifoAnimalShelter/AnimalShelter.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
+package code401Challenges.FifoAnimalShelter;
+
+
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/FifoAnimalShelter/AnimalShelter.java]: Lots of empty lines!
